assets/js/modules/pagespeed-insights/components/FieldReportMetrics.js

```diff
@@ -49,6 +59,21 @@ export default function FieldReportMetrics( { data } ) {
 		/>
 	);
 
+	const footerLink = (
+		<Link
+			href={ addQueryArgs( 'https://developers.google.com/speed/pagespeed/insights/', { url } ) }
+			external
+			inherit
+			dangerouslySetInnerHTML={ sanitizeHTML(
+				_x( 'PageSpeed Insights', 'Service name', 'google-site-kit' ),
+				{
+					ALLOWED_TAGS: [ 'a' ],
+					ALLOWED_ATTR: [ 'href', 'class', 'target' ],
+				}
+			) }
+		/>
+	);
```

Comment:
@aaemnnosttv This was so it wouldn't be rendered in the "Field Data Unavailable" case. It could be kept in `DashboardPageSpeed` and conditionally rendered. But in terms of the HTML/CSS structure it made sense to me to have it in the `FieldReportMetrics` and `LabReportMetrics`.